### PR TITLE
Add BlockedPetBuff

### PR DESCRIPTION
**Pull request by Lyricalpanda**
Have shared logic on pets from charmed and non charmed pets

### CODE REVIEW

File: E3Next/Settings/CharacterSettings.cs [hunk 1]
```diff
@@ -439,7 +440,6 @@ private void LoadData()
             LoadKeyData("Buffs", "Bot Buff", ParsedData, BotBuffs);
             LoadKeyData("Buffs", "Combat Buff", ParsedData, CombatBuffs);
             LoadKeyData("Buffs", "Group Buff", ParsedData, GroupBuffs);
-            LoadKeyData("Buffs", "Pet Buff", ParsedData, PetBuffs);
```
Comment (Lyricalpanda) [inline E3Next/Settings/CharacterSettings.cs:442]: extra line. The other line is visible below in the diff

**Comment by blingeorkl** [comment]
the clicking adds issues if you're on the toon and hit hotkeys like Alt, so 
MQ.Cmd($"/removebuff -pet {buff.SpellName}");
is a better way of executing this

**Comment by Lyricalpanda** [comment]
> the clicking adds issues if you're on the toon and hit keys like Alt when it's trying to click off buffs, so MQ.Cmd($"/removebuff -pet {buff.SpellName}"); is a better way of executing this

Rekka led us astray

**Comment by RekkasGit** [comment]
I agree lets use /removebuff -pet, and i will merge to master

**Comment by Lyricalpanda** [comment]
Done

**Comment by blingeorkl** [comment]
I had something similar to what you have posted, and it was giving me the "I do not have *** that is configured in the bot ini" so I had to add this in BaseSettings.cs under public static void CheckFor

if (sectionkey.Equals("Pets", StringComparison.OrdinalIgnoreCase) && keyData.KeyName.Equals("Blocked Pet Buff", StringComparison.OrdinalIgnoreCase)) return;

**Review by RekkasGit — APPROVED** [review]

**Pull request merged**